src/TheThingsNetwork.cpp

```diff
@@ -251,17 +251,19 @@ int TheThingsNetwork::sendBytes(const byte* buffer, int length, int port, bool c
   str.concat(port);
   if (!sendCommand(str, buffer, length)) {
     debugPrintLn(F("Send command failed"));
-    return 0;
+    return -1;
   }
 
   String response = readLine(10000);
   if (response == "") {
     debugPrintLn(F("Time-out"));
-    return 0;
+    if (confirm == true)
+      return 0;
+    return -2;
```

Comment:
That depends. I'm not sure if without confirm you'd also get this empty response or if it's indeed a separate error. @johanstokking?